contracts/script/deploy/router/CertVerifierRouterDeployer.s.sol

```diff
@@ -26,22 +26,19 @@ contract CertVerifierRouterDeployer is Script, Test {
     // Configuration parameters
     address initialOwner;
     address proxyAdmin;
-    address initialCertVerifier;
+    uint32[] initialRBNs;
+    address[] initialCertVerifiers;
 
     function run(string memory inputJSONFile, string memory outputJSONFile) external {
         // 1. Read the configuration from the JSON input file
         string memory configPath = string.concat("./script/deploy/router/config/", inputJSONFile);
         string memory configData = vm.readFile(configPath);
 
         // Parse configuration parameters
-        bytes memory raw = stdJson.parseRaw(configData, ".initialOwner");
-        initialOwner = abi.decode(raw, (address));
-
-        raw = stdJson.parseRaw(configData, ".initialCertVerifier");
-        initialCertVerifier = abi.decode(raw, (address));
-
-        raw = stdJson.parseRaw(configData, ".proxyAdmin");
-        proxyAdmin = abi.decode(raw, (address));
+        initialOwner = stdJson.readAddress(configData, ".initialOwner");
```

Comment:
Why is the change from `parseRaw` to `readAddress` here? Is this an unrelated change to initializing with multiple addresses?